.ci_support/migrations/linux_python3.8.yaml

```diff
@@ -0,0 +1,26 @@
+c_compiler:
+- gcc
```

Comment:
This won't work without the selectors, cf:https://github.com/conda-forge/py_stringmatching-feedstock/pull/6/files#diff-074883f7d957e193a6bcdd3c93e34b76R23

Maybe revive gh-6 instead?

Reply:
Let's try that. That PR had more issue and typos too.